backend/core/src/listings/listings.service.ts

```diff
@@ -78,8 +96,12 @@ export class ListingsService {
       // (WHERE params are the values passed to andWhere() that TypeORM escapes
       // and substitues for the `:paramName` placeholders in the WHERE clause.)
       .setParameters(innerFilteredQuery.getParameters())
-      .orderBy(getOrderByCondition())
+      .orderBy(getOrderByCondition(params))
+      // Order by unitSummary.unitType.numBedrooms and units.maxOccupancy so that, for a
+      // given listing, its unitSummary's or units are sorted from lowest to highest
+      // bedroom count.
       .addOrderBy("summaryUnitType.num_bedrooms", "ASC", "NULLS LAST")
+      .addOrderBy("units.max_occupancy", "ASC", "NULLS LAST")
```

Comment:
This could have an unexpected sort order if some listings have num_bedrooms set, and some listings have max_occupancy set. Not sure if that's an issue.

Reply:
A thing to note (but I think you already had this in mind): these two `addOrderBy`s are applied last and so won't cause any change in the ordering of two different listings, they'll just affect the ordering of unitSummaries and units _within_ a single listing.

I'm trying to think through ways that this could cause weirdness, and I don't think I see it. If you have one in mind, could you give an example?

(Side note: I'm not sure that sorting the units within a listing makes any difference, since the main (sole?) way we seem to use units data is through `unitsSummarized`, and the units are sorted within `unitsSummarized` when we derive it. But I'm keeping it in here so that there's no behavior change for Exygy.)

Reply:
(Resolving this so that I can submit this and start packaging an upstream PR with these changes, but I would still like to track this so feel free to respond here or in #193 !)

Reply:
I forgot there was already a sort happening so this was only _within_ a listing. My thought had been that if some listings don't have num_bedrooms set, they'd end up with the nulls at the end, and then they'd be sorted by max_occupancy, but you're right that that shouldn't actually matter here.